learn/security/master_api_keys.md

```diff
@@ -158,34 +162,40 @@ We can query our instance to confirm which active keys can search our `patient_m
 {
   "results": [
     {
-      "description": "Default Search API Key (Use it to search from the frontend code)",
-      "key": "0a6e572506c52ab0bd6195921575d23092b7f0c284ab4ac86d12346c33057f99",
+      "name": "Default Search API Key",
+      "description": "Default Search API Key",
```

Comment:
This changed a bit too (ref: https://github.com/meilisearch/specifications/pull/148/files#diff-efbeaa05fe855150edfa47dc218874a3d03dda4000c79375238288abd551694aR77}.

```suggestion
      "description": "Use it to search from the frontend",
```